pkg/operator/nfs/controller.go

```diff
@@ -154,32 +153,58 @@ func createServicePorts() []v1.ServicePort {
 	}
 }
 
-func (c *Controller) createNFSService(nfsServer *nfsServer) error {
+func (c *Controller) createStorageClass(nfsServer *nfsServer) error {
+	for _, export := range nfsServer.spec.Exports {
+		nfsStorageClass := &storage.StorageClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: export.Name,
+				Annotations: map[string]string{
+					"rook.io/nfs-path": "/" + export.PersistentVolumeClaim.ClaimName,
```

Comment:
is an annotation better for this nfs path value than a field in the `parameters` map? if so, why?

Reply:
ahh I think parameter is a better option shifted to `claimPath` key in `parameter`.

Reply:
moved to parameters.